### PR TITLE
Update Twala Testnet (Jellie)

### PR DESCRIPTION
**Pull request by apquinit**

### CODE REVIEW

File: _data/chains/eip155-202624.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
-  "name": "Jellie",
-  "title": "Twala Testnet Jellie",
+  "name": "Twala Testnet",
+  "title": "Twala Testnet (Jellie)",
   "shortName": "twl-jellie",
   "chain": "ETH",
+  "network": "testnet",
```
Comment (ligi) [inline _data/chains/eip155-202624.json:6]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by apquinit** [comment]
JSON updated @ligi

**Comment by apquinit** [comment]
bump @ligi

**Comment by ligi** [comment]
Fix CI errors like this one:
Found file QmTXJVhVKvVC7DQEnGKXvydvwpvVaUEBJrMHvsCr4nr1sK in ../_data/iconsDownload that is not referenced

**Pull request closed without merging**